policy/modules/services/certbot.fc

```diff
@@ -2,3 +2,9 @@
 /usr/bin/letsencrypt	--	gen_context(system_u:object_r:certbot_exec_t,s0)
 /var/lib/letsencrypt(/.*)?	gen_context(system_u:object_r:certbot_lib_t,s0)
 /var/log/letsencrypt(/.*)?	gen_context(system_u:object_r:certbot_log_t,s0)
+
+/etc/acme-sh(/.*)?		gen_context(system_u:object_r:certbot_lib_t,s0)
+/usr/share/acme\.sh(/.*)?		gen_context(system_u:object_r:certbot_lib_t,s0)
+/usr/share/acme\.sh/acme\.sh	--	gen_context(system_u:object_r:certbot_exec_t,s0)
+/root/\.acme\.sh(/.*)?		gen_context(system_u:object_r:certbot_lib_t,s0)
+/root/\.acme\.sh/acme\.sh	--	gen_context(system_u:object_r:certbot_exec_t,s0)
```

Comment:
I would probably not support this "/root" scenario

Reply:
Ok.